### ISSUE TITLE
Setting unknown property: craft\elements\MatrixBlock::COUNT(*) is thrown on front-end entry submission

### ISSUE THREAD

**Issue by Saboteur777** [craftcms/cms#5690]
### Description

`Unknown Property – yii\base\UnknownPropertyException
Setting unknown property: craft\elements\MatrixBlock::COUNT(*)` is thrown when I try to save an entry with the following entry form: 

```
<form method="post" accept-charset="UTF-8">
    {{ csrfInput() }}
    {{ actionInput('entries/save-entry') }}
    {{ redirectInput(url('site-added')) }}
    {{ hiddenInput('sectionId', '1') }}
    {{ hiddenInput('enabled', '1') }}

    <div class="flex flex-wrap">
        <div class="w-full md:w-1/2 mb-4 px-2">
            {% include '_input/input.twig' with {
                'label': 'Site Name' | t,
                'type': 'text',
                'fieldHandle': 'fields[sitesName]',
                'options': {
                    'value': entry.sitesName ?? '',
                    'required': true,
                    'placeholder': 'Craft Report backend',
                }
            } %}
        </div>

        <div class="w-full md:w-1/2 mb-4 px-2">
            {% set options = [
                {
                    'value': 'production',
                    'label': 'Production'
                },
                {
                    'value': 'staging',
                    'label': 'Staging'
                },
                {
                    'value': 'development',
                    'label': 'Development'
                }
            ] %}

            {% include '_input/select.twig' with {
                'label': 'Environment' | t,
                'type': 'text',
                'fieldHandle': 'fields[sitesEnvironment]',
                'options': {
                    'value': entry.sitesEnvironment ?? '',
                    'required': true,
                    'selectOptions': options
                }
            } %}
        </div>

        <div class="w-full md:w-1/2 mb-4 px-2">
            {% include '_input/input.twig' with {
                'label': 'Site URL' | t,
                'type': 'text',
                'fieldHandle': 'fields[sitesUrl]',
                'options': {
                    'value': entry.sitesUrl ?? '',
                    'required': true,
                    'placeholder': 'https://app.craft.report',
                }
            } %}
        </div>

        <div class="w-full md:w-1/2 mb-4 px-2">
            {% include '_input/input.twig' with {
                'label': 'Site API Key' | t,
                'type': 'text',
                'fieldHandle': 'fields[sitesApiKey]',
                'options': {
                    'value': entry.sitesApiKey ?? '',
                    'required': true,
                    'placeholder': '1qay2wsx3edc4rfv5tgb6zhn ',
                }
            } %}
        </div>

        <div class="w-full flex flex-wrap items-center">
            <input type="checkbox" class="checkbox toggle-div-coming-after-this hidden" id="httpAuth">
            <label for="httpAuth" class="w-full order-2 pl-2 text-gray-400 text-sm hover:text-gray-500 transition-default">
                HTTP Authentication details
            </label>

            <div class="http-auth w-full flex-1 flex flex-wrap order-3">
                <div class="w-full md:w-1/2 mb-4 px-2">
                    {% include '_input/input.twig' with {
                        'label': 'Auth Username' | t,
                        'type': 'text',
                        'fieldHandle': 'fields[authUsername]',
                        'options': {
                            'value': entry.authUsername ?? ''
                        }
                    } %}
                </div>

                <div class="w-full md:w-1/2 mb-4 px-2">
                    {% include '_input/input.twig' with {
                        'label': 'Auth Password' | t,
                        'type': 'text',
                        'fieldHandle': 'fields[authPassword]',
                        'options': {
                            'value': entry.authPassword ?? ''
                        }
                    } %}
                </div>
            </div>
        </div>

        <div class="w-full flex flex-wrap items-center">
            <input type="checkbox" class="checkbox toggle-div-coming-after-this hidden" id="notificationSettings">
            <label for="notificationSettings" class="w-full order-2 pl-2 text-gray-400 text-sm hover:text-gray-500 transition-default">
                Notification preferences
            </label>

            <div class="http-auth w-full flex-1 flex flex-wrap order-3">
                <div class="w-full md:w-1/2 mb-4 px-2">
                    {% set options = [
                        {
                            'value': 'all',
                            'label': 'All'
                        },
                        {
                            'value': 'criticalOnly',
                            'label': 'Critical Only'
                        },
                        {
                            'value': 'none',
                            'label': 'None'
                        }
                    ] %}

                    {% include '_input/select.twig' with {
                        'label': 'Update notifications' | t,
                        'type': 'text',
                        'fieldHandle': 'fields[updateNotifications]',
                        'options': {
                            'value': entry.updateNotifications ?? '',
                            'required': true,
                            'selectOptions': options
                        }
                    } %}
                </div>

                <div class="w-full md:w-1/2 mb-4 px-2">
                    {% set options = [
                        {
                            'value': 1,
                            'label': 'Yes'
                        },
                        {
                            'value': 0,
                            'label': 'No'
                        }
                    ] %}

                    {% include '_input/select.twig' with {
                        'label': 'Licence issues notifications' | t,
                        'type': 'text',
                        'fieldHandle': 'fields[licenceIssueNotifications]',
                        'options': {
                            'value': true,
                            'required': true,
                            'selectOptions': options
                        }
                    } %}
                </div>
            </div>
        </div>
    </div>

    <div class="px-2 text-center md:text-right">
        {% include '_input/submit.twig' with {
            'text': 'Add site',
            'variant': 'is-primary'
        } %}
    </div>
</form>
```

Saving entries from the CP does not throw this error. What is wrong with this form? 

### Additional info

- Craft version: 3.4.6.1
- PHP version: 7.4.2
- Database driver & version: MySQL 5.5.5
- Plugins & versions:
Bugsnag 2.0.6
CP Field Inspect    1.1.2
Craft Reporter  1.5.4
Environment Label   3.2.0
Imager  v2.4.0
Mailgun 1.4.2
Notifications   1.1.1
Redactor    2.6.0.1
Support dev-master
Twigpack    1.1.14
Twig Profiler   1.0.1

**Comment by brandonkelly** [comment]
This is covered by #5618, which is fixed for the next release.

**Comment by brandonkelly** [comment]
Actually this bug was just tacked onto that issue, so I’ll point the bugfix release note to this issue.

**Comment by Saboteur777** [comment]
@brandonkelly Thanks, using this in my `composer.json` fixed it: 

```
"craftcms/cms": "dev-develop#a49de72afcf18349617bff20b82c7c197218af42 as 3.4.6.1",
```

**Comment by brandonkelly** [comment]
Just released 3.4.7 with the fix.

**Comment by Saboteur777** [comment]
Thanks!
